index.html

```diff
@@ -54,6 +54,17 @@ <h2 class="sr-only">Results</h2>
                 <h3 class="text-small">Contrast Ratio</h3>
                 <p id="contrast-ratio-result" class="large"></p>
               </div>
+              <div class="error-message">
+                <h2>
+                  Invalid input
+                </h2>
+                <p>The following input formats are supported
```

Comment:
If your intention was to list all supported formats, then maybe RGB format is needed here?

Reply:
It definitely felt like a jumped the gun when I saw the new issues opened by Jessica.
>some logic needed in order to display error message not too early. to implement this for all formats at once - could be hard
yep that's a Tricky one, already did battle with it making this issue but thinking about it now with a single color input in mind I think I got just the thing for it.
I will get everything and push hopefully before the day runs out.